posthog/models/error_tracking/error_tracking.py

```diff
@@ -58,3 +58,15 @@ def convert_fingerprints_to_tuples(fps: list[list[str]]):
         # converting back to list of lists before saving
         self.merged_fingerprints = [list(f) for f in merged_fingerprints]
         self.save()
+
+
+class ErrorTrackingIssueFingerprint(models.Model):
+    team = models.ForeignKey("Team", on_delete=models.CASCADE, db_index=False)
+    issue = models.ForeignKey(ErrorTrackingGroup, on_delete=models.CASCADE)
+    fingerprint = models.CharField(max_length=400)
+
+    # current version of the id, used to sync with ClickHouse and collapse rows correctly for overrides ClickHouse table
+    version = models.BigIntegerField(null=True, blank=True)
```

Comment:
This probably shouldn't be nullable (we always want these records to be versioned.)

Reply:
Note to self: don't [copy persons](https://github.com/PostHog/posthog/blob/f31abb5787f7c516bafcd515abe65b68b86a3cba/posthog/models/person/person.py#L126) so directly 😂